js/src/tests/run_trees.int.test.ts

```diff
@@ -214,3 +214,31 @@ test.concurrent(
   },
   120_000
 );
+
+test.concurrent(
+  "Test end() write to metadata",
+  async () => {
+    const runMeta = uuid.v4();
+    const projectName = `__test_end_metadata_run_tree_js ${runMeta}`;
```

Comment:
If you're creating a project with UUID in the name, you should clean it up after the test. Also should consolidate how you're doing in py and JS. I would honestly just create a unique project and clean up at end of test in py and js